src/xo-app/navbar/index.js

```diff
@@ -0,0 +1,42 @@
+import Icon from 'icon'
+import React, { Component } from 'react'
+import {
+  connectStore,
+  propTypes,
+  autobind
+} from 'utils'
+
+@connectStore([
+  'user'
+])
+@propTypes({
+  selectLang: propTypes.func.isRequired
+})
+export default class Navbar extends Component {
+  @autobind
+  handleSelectLang (event) {
+    // FIXME: find a way to reach selectLang to set the app language
+    this.props.selectLang(event.target.value)
+  }
+  render () {
+    const {
+      user
+    } = this.props
+    return <nav className='navbar navbar-full navbar-light bg-faded xo-navbar'>
+      <a style={{minWidth: 'max-content'}} className='navbar-brand xo-brand' href='#'>Xen Orchestra</a>
```

Comment:
Respect as much as possible Bootstrap architecture.

And remove AMAP custom styles.